WordPress/Classes/ViewRelated/Blog/Blog Details/Detail Header/NewBlogDetailHeaderView.swift

```diff
@@ -167,7 +167,7 @@ class NewBlogDetailHeaderView: UIView, BlogDetailHeader {
     }
 
     @objc private func titleButtonTapped() {
-        QuickStartTourGuide.find()?.visited(.siteTitle)
```

Comment:
Sorry a bit late, so I don't expect this to be changed in this PR.  But is there a case for keeping the `.find()` method around?

Maybe it's best to just remove it?

Reply:
Oh nm, I just realized the method was removed and this was probably the result of our PRs being merged.